client/apps/game/src/ui/modules/entity-details/hex-entity-details.tsx

```
@@ -0,0 +1,105 @@
+        />
+      </div>
+    );
+  }
+
+  const biome = useMemo(() => {
+    return Biome.getBiome(selectedHex?.col || 0, selectedHex?.row || 0);
+  }, [selectedHex]);
+
+  return (
+    <div className="h-full overflow-auto">
+      <div className={`${!!entityContent ? "h-50" : "h-full"}`}>
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)